### PR TITLE
LPS-114909 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -11,13 +11,14 @@
...
 
-			<!-- Enable this profile in standalone -->
+			<!-- Uncomment the following element if project is not in Workspace
 
-			<id>standalone</id>
 			<activation>
-				<activeByDefault>false</activeByDefault>
-			</activation>
+				<activeByDefault>true</activeByDefault>
+				<
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml:20]: Is this a typo?

File: modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -11,13 +11,14 @@
 	<version>${version}</version>
 	<profiles>
 		<profile>
+			<id>standalone</id>
 
-			<!-- Enable this profile in standalone -->
+			<!-- Uncomment the following element if project is not in Workspace
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml:16]: Please update to " ... if the project is not in workspace ... "

File: modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml:20]: That is correct. We needed the switch the logic around.

File: modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml (hunk 2 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml:16]: Okay. I'll resend

**Comment by lawrence-lee** [comment]
ci:stop

File: modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml:20]: Nm! i see the weird `<`. Removing

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by petershin** [comment]
ci:forward

**Pull request closed without merging**